### PR TITLE
Saving and retrieving block data with the database was added

### PR DESCRIPTION
**Pull request by brianlee389**
The functionality for adding, retrieving, and updating block data to the database was implemented.
Also the saved block data is repopulated onto admin page whenever the site is visited to see the saved changes.

### CODE REVIEW

**Comment by brianlee389** [comment]
I forgot to change code before I committed. I forgot to make the update/publish button save the block data.

**Comment by thomaspark** [comment]
Looks good, merged. Just a couple of inline comments, and will commit formatting fixes.

File: scripts/snowball-ajax.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	$("#publish").click(function() {
+		var blocksRetrieved = retrieveBlocks();
+
+		var data = {
+			'action': 'add_blocks',		// this is needed to know which callback to use
+			'blocks': blocksRetrieved,
+			'post_id' : ajax_object.post_id
+		};
+		$.post(ajax_object.ajax_url, data, function(response) {
+			console.log("The response from clicking the text button: " + response);
```
Comment (thomaspark) [inline scripts/snowball-ajax.js:12]: do we need to keep this in?

File: snowball.php [hunk 2]
```diff
@@ -171,4 +171,151 @@ function snowball_metabox_save($post_id) {
...
+      current_time('mysql'), $post_id, $json_block, current_time('mysql'), $json_block);
+    $was_successful = $wpdb->query($prepared_query);
+  */
+  $was_updated = $wpdb->update( $table_name, 
+    array( 
+      'time' => current_time( 'mysql' ), 
+      'blocks_value' => $json_block,
+      ),
+    array( "post_id" => $post_id), 
+    array("%s", "%s", ),
```
Comment (thomaspark) [inline snowball.php:278]: this looks unfinished?

**Pull request merged**